resources/views/mini_views/navigational_view.blade.php

```diff
@@ -1,17 +1,23 @@
+<style>
+    .tab-content {
```

Comment:
do not override default css of the theme here. It will effect all `tab-content` on the page

Reply:
tab-content border making as hidden so that the border doesn't looks odd inside box content. It is intended for all tab-contents on this page.